qemu_v8.mk

```diff
@@ -498,6 +514,36 @@ else
 QEMU_MTE	= off
 endif
 
+QEMU_ARGS = -nographic
+QEMU_ARGS += -smp $(QEMU_SMP)
+QEMU_ARGS += -cpu $(QEMU_CPU)
+QEMU_ARGS += -d unimp -semihosting-config enable=on,target=native
+QEMU_ARGS += -m $(QEMU_MEM)
+QEMU_ARGS += -bios bl1.bin
+QEMU_ARGS += -initrd rootfs.cpio.gz
+QEMU_ARGS += -kernel Image
+QEMU_ARGS += -append 'console=ttyAMA0,38400 keep_bootcon root=/dev/vda2 $(QEMU_KERNEL_BOOTARGS)'
+QEMU_ARGS += $(QEMU_XEN)
+QEMU_ARGS += $(QEMU_EXTRA_ARGS)
+QEMU_ARGS += -machine virt,acpi=off,secure=on,mte=$(QEMU_MTE),gic-version=$(QEMU_GIC_VERSION),virtualization=$(QEMU_VIRT)
+
+ifeq ($(WITH_SCMI),y)
+QEMU_DUMP_ARGS = $(QEMU_ARGS),dumpdtb=qemu_v8.dtb
```

Comment:
Appending `,dumpdtb=qemu_v8.dtb` is a bit fragile. I propose getting rid of `QEMU_DUMP_ARGS` and use `$`(QEMU_ARGS) -machine dumpdtb=qemu_v8.dtb` in the `$(SCMI_DTB)` target below.

Reply:
fair enough